.github/workflows/schedule.yml

```diff
@@ -0,0 +1,150 @@
+name: SCHEDULED
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 16 ? * 4"
```

Comment:
What's meaning of the `0 16 ? * 4`, plz add some comments for it :)

Reply:
done